lib/zinnia.js

```diff
@@ -0,0 +1,144 @@
+import timers from 'node:timers/promises'
+import { execa } from 'execa'
+import * as Sentry from '@sentry/node'
+import { installBinaryModule, downloadSourceFiles, getBinaryModuleExecutable } from './modules.js'
+import { moduleBinaries } from './paths.js'
+
+const ZINNIA_DIST_TAG = 'v0.7.0'
+const ZINNIA_MODULES = [
+  {
+    module: 'peer-checker',
+    repo: 'github.com/filecoin-station/mod-peer-checker',
```

Comment:
The `downlaodSourceFiles` method depends on public GitHub APIs, so I think we can bake that assumption into the arguments by changing the repo string to just `filecoin-station/mod-peer-checker`

Reply:
Great idea. Addressed in f85c4a6